packages/ilios-common/addon/components/course/objective-list-item.js

```diff
@@ -67,22 +67,22 @@ export default class CourseObjectiveListItemComponent extends Component {
       return [...set, ...cohortObjectives.flat()];
     }, []);
     const parents = await this.args.courseObjective.programYearObjectives;
-    this.parentsBuffer = parents.slice().map((objective) => {
+    this.parentsBuffer = parents.map((objective) => {
       return findById(objectives, objective.id);
     });
     this.isManagingParents = true;
   });
 
   manageDescriptors = dropTask(async () => {
     const meshDescriptors = await this.args.courseObjective.meshDescriptors;
-    this.descriptorsBuffer = meshDescriptors.slice();
```

Comment:
The entire purpose of this interim variable seems to be the slice. This might be the only time we notice this for a while so please replace with a direct assignment eg `this.descriptorsBuffer = await this.args.courseObjective.meshDescriptors;`